packages/webapp/src/components/Task/FieldWorkTask/index.jsx

```diff
@@ -41,6 +36,31 @@ const PureFieldWorkTask = ({
   useEffect(() => {
     if (fieldWorkTypeExposedValue?.value !== 'OTHER') setValue(FIELD_WORK_OTHER_TYPE, null);
   }, [fieldWorkTypeExposedValue]);
+
+  useEffect(() => {
+    setFieldWorkTypeOptions((allOptions) => {
+      let options = fieldWorkTypes.map((f) =>
+        f.farm_id === null ? { ...f, label: t(f.label) } : { ...f, label: f.field_work_name },
+      );
+      options.push({ label: t('ADD_TASK.FIELD_WORK_VIEW.TYPE.OTHER'), value: 'OTHER' });
+      return options;
+    });
+  }, [fieldWorkTypes]);
+
+  useEffect(() => {
+    dispatch(getFieldWorkTypes());
+  }, []);
+
+  const displayLabel = (value) => {
+    const translationKey = `ADD_TASK.FIELD_WORK_VIEW.TYPE.${value}`;
```

Comment:
types are usually in their only translation files like crop, then this can be t(translationKey)

Reply:
Didn’t get you